README.md

```diff
@@ -1,6 +1,10 @@
-Zcash 1.0.0
+Zcash Classic 1.0.0
 ===========
```

Comment:
Well, be that as it may, since this is the beginning of their ponzi, they can afford to pay lawyers to enforce the trademark... you know... since paying old investors with new investor money is relatively easy when a ponzi first begins. Right @daira ?